ui/src/components/AppsTable.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
 import { withRouter } from 'react-router-dom';
 
-export const AppsTable = ({ columns, rows, sortBy, onSort, onRowClick }) => (
-  <Table aria-label="Mobile Apps" sortBy={sortBy} onSort={onSort} cells={columns} rows={rows}>
+export const AppsTable = ({ caption, columns, rows, sortBy, onSort, onRowClick }) => (
+  <Table aria-label="Mobile Apps" caption={caption} sortBy={sortBy} onSort={onSort} cells={columns} rows={rows}>
```

Comment:
@damienomurchu I don't think we need to use caption here. I removed it from the other table previously as it wasn't showing up very well. I've used a Title component instead which you can see here: https://github.com/aerogear/mobile-security-service/pull/94/files#diff-eced989d8adfc27e10bff4535f0c7a97

Reply:
yes, I saw that, and think a title component will be more straightforward :+1: